website/docs/language/settings/backends/configuration.mdx

```diff
@@ -71,18 +71,20 @@ If a configuration includes no backend block, Terraform defaults to using the `l
 
 ## Initialization
 
-Whenever a configuration's backend changes, you must run `terraform init` again
+When you change a backend's configuration, you must run `terraform init` again
 to validate and configure the backend before you can perform any plans, applies,
 or state operations.
 
-When changing backends, Terraform will give you the option to migrate
+After you initialize, Terraform creates `.terraform/terraform.tfstate` locally. This file contains the most recent backend configuration, including any authentication parameters you provided to the Terraform CLI. Do not check `.terraform/terraform.tfstate` into Git, as it may contain sensitive credentials for your remote backend.
```

Comment:
```suggestion
After you initialize, Terraform creates a `.terraform/` directory locally. This directory contains the most recent backend configuration, including any authentication parameters you provided to the Terraform CLI. Do not check this directory into Git, as it may contain sensitive credentials for your remote backend.
```